### DISCUSSION TITLE
glb not able call/load from external link

### DISCUSSION THREAD

**Discussion by Arumugamhns** [google/model-viewer#4173] [Q&A]
I have call the src dynamically from api..

so the src path is 
https://www.domainname.in/assets/Resources/models/sample.glb

but i cannot able to show this on browser i experinece the attached error. but .glb file call from dropbox its working. 

is there any settings for access the file from our server

![image](https://user-images.githubusercontent.com/60291301/226343564-c923beeb-21da-41dd-a1a6-2c75b4a6b24c.png)

**Comment by elalish** [comment]
Sounds like a server settings problem. Maybe you need to change what domains are allowed access to your server?
